exercises/concept/dancing-dots/.docs/instructions.md

```diff
@@ -35,7 +35,7 @@ Use the `Animation` behaviour to implement a flickering animation.
 
 It should use the default `init/1` callback because it doesn't take any options.
 
-Implement the `handle_frame/3` callback. In every 4th frame, it should return the dot with half of its original opacity. In other frames, it should return the dot unchanged.
+Implement the `handle_frame/3` callback. This function handles a single frame. If the frame is a multiple of four, the function should return the dot with half of its original opacity. In other frames, it should return the dot unchanged.
```

Comment:
```suggestion
Implement the `handle_frame/3` callback, which handles a single frame at a time. If the frame number is a multiple of four, the function should return the dot with half of its original opacity. In other frames, it should return the dot unchanged.
```

Reply:
I think the addition of "at a time" might reintroduce the same confusion about whether the function handles a single frame vs processing each frame and returning a dot or a halved dot. But obviously other people figured it out, so it might just be me.

Reply:
OK, let's remove at a time. The goal is to be as clear as possible :)
```suggestion
Implement the `handle_frame/3` callback, which handles a single frame. If the frame number is a multiple of four, the function should return the dot with half of its original opacity. In other frames, it should return the dot unchanged.
```